From 1e849c3e0704dc4688f3517b010262d0cbbda726 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Mon, 26 Feb 2024 19:46:38 +0100
Subject: [PATCH 0690/1695] dts: stm32: secure lptimer1 for clock calibration
 on stm32mp257f-dk

This LPTIMER is used for clock calibrations in the secure world.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git e38eadbfc44b67b09594467f187e1510a227154c dts: stm32: secure lptimer1 for clock calibration on stm32mp257f-dk]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: Ie6831bc9a20aee0b247ecc8a6ef22bd0f850b8ff
---
 core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rif.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rif.dtsi b/core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rif.dtsi
index bb336213d..1346cb6fa 100644
--- a/core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rif.dtsi
+++ b/core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rif.dtsi
@@ -22,7 +22,7 @@
 		RIFPROT(STM32MP25_RIFSC_TIM16_ID, RIF_UNUSED, RIF_UNLOCK, RIF_NSEC, RIF_NPRIV, RIF_UNUSED, RIF_SEM_DIS, RIF_CFDIS)
 		RIFPROT(STM32MP25_RIFSC_TIM17_ID, RIF_UNUSED, RIF_UNLOCK, RIF_NSEC, RIF_NPRIV, RIF_UNUSED, RIF_SEM_DIS, RIF_CFDIS)
 		RIFPROT(STM32MP25_RIFSC_TIM20_ID, RIF_UNUSED, RIF_UNLOCK, RIF_NSEC, RIF_NPRIV, RIF_UNUSED, RIF_SEM_DIS, RIF_CFDIS)
-		RIFPROT(STM32MP25_RIFSC_LPTIM1_ID, RIF_UNUSED, RIF_UNLOCK, RIF_NSEC, RIF_NPRIV, RIF_CID1, RIF_SEM_DIS, RIF_CFEN)
+		RIFPROT(STM32MP25_RIFSC_LPTIM1_ID, RIF_UNUSED, RIF_UNLOCK, RIF_SEC, RIF_PRIV, RIF_CID1, RIF_SEM_DIS, RIF_CFEN)
 		RIFPROT(STM32MP25_RIFSC_LPTIM2_ID, RIF_UNUSED, RIF_UNLOCK, RIF_NSEC, RIF_NPRIV, RIF_UNUSED, RIF_SEM_DIS, RIF_CFDIS)
 		RIFPROT(STM32MP25_RIFSC_LPTIM3_ID, RIF_UNUSED, RIF_UNLOCK, RIF_NSEC, RIF_NPRIV, RIF_UNUSED, RIF_SEM_DIS, RIF_CFDIS)
 		RIFPROT(STM32MP25_RIFSC_LPTIM4_ID, RIF_UNUSED, RIF_UNLOCK, RIF_NSEC, RIF_NPRIV, RIF_UNUSED, RIF_SEM_DIS, RIF_CFDIS)
-- 
2.47.2

